Can't build ios without some modules

##### System information (version)
- OpenCV => 4.4.x (master)
- Operating System / Platform => MAC Catalina
- Compiler => Cmake 3.8
- Python => 2.7
- Xcode => 11.5

##### Detailed description

I'm using [platforms/ios/build_framework.py](https://github.com/opencv/opencv/blob/master/platforms/ios/build_framework.py) to build opencv for ios swift project.
When I run the script(build_framework.py) without any arguments, it works well.
But if I put arguments **--without**, the build fails.

I checked this issue in Xcode 11.5, 11.7, 12.0 and all the same result.
The error says
```
CMake Error at modules/objc/generator/CMakeLists.txt:86 (add_dependencies):
Cannot add target-level dependencies to non-existent target "opencv_world".
```

This [line](https://github.com/opencv/opencv/blob/f7b8f522ff4084d584628811a02a621736c3a731/platforms/ios/build_framework.py#L186) of code makes change if I use argument **--without**.

##### Steps to reproduce

```bash
python platforms/ios/build_framework.py build_ios_without_doublecheck_2 --without calib3d
```

##### Issue submission checklist

 - [x] I report the issue, it's not a question
   <!--
   OpenCV team works with answers.opencv.org, Stack Overflow and other communities
   to discuss problems. Tickets with question without real issue statement will be
   closed.
   -->
 - [x] I checked the problem with documentation, FAQ, open issues,
       answers.opencv.org, Stack Overflow, etc and have not found solution
   <!--
   Places to check:
   * OpenCV documentation: https://docs.opencv.org
   * FAQ page: https://github.com/opencv/opencv/wiki/FAQ
   * OpenCV forum: https://answers.opencv.org
   * OpenCV issue tracker: https://github.com/opencv/opencv/issues?q=is%3Aissue
   * Stack Overflow branch: https://stackoverflow.com/questions/tagged/opencv
   -->
 - [x] I updated to latest OpenCV version and the issue is still there
   <!--
   master branch for OpenCV 4.x and 3.4 branch for OpenCV 3.x releases.
   OpenCV team supports only latest release for each branch.
   The ticket is closed, if the problem is not reproduced with modern version.
   -->
 - [x] There is reproducer code and related data files: videos, images, onnx, etc
   <!--
   The best reproducer -- test case for OpenCV that we can add to the library.
   Recommendations for media files and binary files:
   * Try to reproduce the issue with images and videos in opencv_extra repository
     to reduce attachment size
   * Use PNG for images, if you report some CV related bug, but not image reader
     issue
   * Attach the image as archive to the ticket, if you report some reader issue.
     Image hosting services compress images and it breaks the repro code.
   * Provide ONNX file for some public model or ONNX file with with random weights,
     if you report ONNX parsing or handling issue. Architecture details diagram
     from netron tool can be very useful too. See https://lutzroeder.github.io/netron/
   -->

--- a/modules/objc/generator/CMakeLists.txt
+++ b/modules/objc/generator/CMakeLists.txt
@@ -77,10 +77,8 @@ add_custom_command(
     COMMENT "Generate files for Objective-C bindings"
 )
 
-add_custom_target(gen_opencv_objc_source DEPENDS ${objc_generated_files}
+add_custom_target(gen_opencv_objc_source ALL DEPENDS ${objc_generated_files}
     SOURCES "${OBJC_SOURCE_DIR}/generator/gen_objc.py"
             "${OBJC_SOURCE_DIR}/generator/templates/cmakelists.template"
             "${CMAKE_CURRENT_BINARY_DIR}/gen_objc.json"
 )
-
-add_dependencies(opencv_world gen_opencv_objc_source)

--- a/modules/objc/test/cmakelists.template
+++ b/modules/objc/test/cmakelists.template
@@ -27,6 +27,7 @@ xctest_add_bundle(OpenCVTestTests OpenCVTest ${test_sources} test/resources ${FR
 set_target_properties(OpenCVTestTests PROPERTIES
     XCODE_ATTRIBUTE_FRAMEWORK_SEARCH_PATHS "${FRAMEWORK_DIR}"
     XCODE_ATTRIBUTE_OTHER_SWIFT_FLAGS "-Xcc ${SUPPRESS_WARNINGS_FLAGS}"
+    XCODE_ATTRIBUTE_LD_RUNPATH_SEARCH_PATHS "${FRAMEWORK_DIR}"
     )
 
 # link necessary Frameworks

--- a/platforms/ios/build_framework.py
+++ b/platforms/ios/build_framework.py
@@ -295,7 +295,7 @@ class Builder:
             "-Xlinker", "/usr/lib/swift",
             "-target", link_target,
             "-isysroot", sdk_dir,
-            "-install_name", ("@executable_path/Frameworks/" + self.framework_name + ".framework/" + self.framework_name) if is_device else res,
+            "-install_name", "@rpath/" + self.framework_name + ".framework/" + self.framework_name,
             "-dynamiclib", "-dead_strip", "-fobjc-link-runtime", "-all_load",
             "-o", res
         ] + swift_link_dirs + bitcode_flags + module + libs + libs3)

--- a/samples/swift/ios/ColorBlobDetection/ColorBlobDetectionDynamic.xcodeproj/project.pbxproj
+++ b/samples/swift/ios/ColorBlobDetection/ColorBlobDetectionDynamic.xcodeproj/project.pbxproj
@@ -1,0 +1,413 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 50;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		5B8C4C2D243814E9003D9176 /* libc++.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = 5B8C4C2C243814A0003D9176 /* libc++.tbd */; };
+		5B8C4C31243849E9003D9176 /* ColorBlobDetector.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5B8C4C30243849E9003D9176 /* ColorBlobDetector.swift */; };
+		5BBDB48A240D380F00BB10FE /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5BBDB489240D380F00BB10FE /* AppDelegate.swift */; };
+		5BBDB48E240D380F00BB10FE /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5BBDB48D240D380F00BB10FE /* ViewController.swift */; };
+		5BBDB491240D380F00BB10FE /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 5BBDB48F240D380F00BB10FE /* Main.storyboard */; };
+		5BBDB493240D381000BB10FE /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 5BBDB492240D381000BB10FE /* Assets.xcassets */; };
+		5BBDB496240D381000BB10FE /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 5BBDB494240D381000BB10FE /* LaunchScreen.storyboard */; };
+		5BBDB517241923C800BB10FE /* AVFoundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5BBDB516241923C800BB10FE /* AVFoundation.framework */; };
+		5BBDB5192419241200BB10FE /* CoreMedia.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5BBDB5182419241200BB10FE /* CoreMedia.framework */; };
+		5BE7205625201342004E84AE /* OpenCV.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5B14CF7F2452E926003438BE /* OpenCV.framework */; };
+		5BE7205725201342004E84AE /* OpenCV.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 5B14CF7F2452E926003438BE /* OpenCV.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+/* End PBXBuildFile section */
+
+/* Begin PBXCopyFilesBuildPhase section */
+		5BE7205825201342004E84AE /* Embed Frameworks */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				5BE7205725201342004E84AE /* OpenCV.framework in Embed Frameworks */,
+			);
+			name = "Embed Frameworks";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXCopyFilesBuildPhase section */
+
+/* Begin PBXFileReference section */
+		5B14CF7F2452E926003438BE /* OpenCV.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = OpenCV.framework; path = ../../../OpenCV.framework; sourceTree = "<group>"; };
+		5B8C4C2C243814A0003D9176 /* libc++.tbd */ = {isa = PBXFileReference; lastKnownFileType = "sourcecode.text-based-dylib-definition"; name = "libc++.tbd"; path = "usr/lib/libc++.tbd"; sourceTree = SDKROOT; };
+		5B8C4C30243849E9003D9176 /* ColorBlobDetector.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ColorBlobDetector.swift; sourceTree = "<group>"; };
+		5BBDB486240D380F00BB10FE /* ColorBlobDetection.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = ColorBlobDetection.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		5BBDB489240D380F00BB10FE /* AppDelegate.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AppDelegate.swift; sourceTree = "<group>"; };
+		5BBDB48D240D380F00BB10FE /* ViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ViewController.swift; sourceTree = "<group>"; };
+		5BBDB490240D380F00BB10FE /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/Main.storyboard; sourceTree = "<group>"; };
+		5BBDB492240D381000BB10FE /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
+		5BBDB495240D381000BB10FE /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
+		5BBDB497240D381000BB10FE /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
+		5BBDB516241923C800BB10FE /* AVFoundation.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = AVFoundation.framework; path = System/Library/Frameworks/AVFoundation.framework; sourceTree = SDKROOT; };
+		5BBDB5182419241200BB10FE /* CoreMedia.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = CoreMedia.framework; path = System/Library/Frameworks/CoreMedia.framework; sourceTree = SDKROOT; };
+/* End PBXFileReference section */
+
+/* Begin PBXFrameworksBuildPhase section */
+		5BBDB483240D380F00BB10FE /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				5BBDB5192419241200BB10FE /* CoreMedia.framework in Frameworks */,
+				5B8C4C2D243814E9003D9176 /* libc++.tbd in Frameworks */,
+				5BBDB517241923C800BB10FE /* AVFoundation.framework in Frameworks */,
+				5BE7205625201342004E84AE /* OpenCV.framework in Frameworks */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXFrameworksBuildPhase section */
+
+/* Begin PBXGroup section */
+		5BBDB47D240D380F00BB10FE = {
+			isa = PBXGroup;
+			children = (
+				5BBDB488240D380F00BB10FE /* ColorBlobDetection */,
+				5BBDB487240D380F00BB10FE /* Products */,
+				5BBDB515241923C800BB10FE /* Frameworks */,
+			);
+			sourceTree = "<group>";
+		};
+		5BBDB487240D380F00BB10FE /* Products */ = {
+			isa = PBXGroup;
+			children = (
+				5BBDB486240D380F00BB10FE /* ColorBlobDetection.app */,
+			);
+			name = Products;
+			sourceTree = "<group>";
+		};
+		5BBDB488240D380F00BB10FE /* ColorBlobDetection */ = {
+			isa = PBXGroup;
+			children = (
+				5B14CF7F2452E926003438BE /* OpenCV.framework */,
+				5BBDB489240D380F00BB10FE /* AppDelegate.swift */,
+				5BBDB48D240D380F00BB10FE /* ViewController.swift */,
+				5BBDB48F240D380F00BB10FE /* Main.storyboard */,
+				5BBDB492240D381000BB10FE /* Assets.xcassets */,
+				5BBDB494240D381000BB10FE /* LaunchScreen.storyboard */,
+				5BBDB497240D381000BB10FE /* Info.plist */,
+				5B8C4C30243849E9003D9176 /* ColorBlobDetector.swift */,
+			);
+			path = ColorBlobDetection;
+			sourceTree = "<group>";
+		};
+		5BBDB515241923C800BB10FE /* Frameworks */ = {
+			isa = PBXGroup;
+			children = (
+				5B8C4C2C243814A0003D9176 /* libc++.tbd */,
+				5BBDB5182419241200BB10FE /* CoreMedia.framework */,
+				5BBDB516241923C800BB10FE /* AVFoundation.framework */,
+			);
+			name = Frameworks;
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXNativeTarget section */
+		5BBDB485240D380F00BB10FE /* ColorBlobDetection */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 5BBDB4B0240D381000BB10FE /* Build configuration list for PBXNativeTarget "ColorBlobDetection" */;
+			buildPhases = (
+				5BBDB482240D380F00BB10FE /* Sources */,
+				5BBDB483240D380F00BB10FE /* Frameworks */,
+				5BBDB484240D380F00BB10FE /* Resources */,
+				5BE7205825201342004E84AE /* Embed Frameworks */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = ColorBlobDetection;
+			productName = ColorBlobDetection;
+			productReference = 5BBDB486240D380F00BB10FE /* ColorBlobDetection.app */;
+			productType = "com.apple.product-type.application";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		5BBDB47E240D380F00BB10FE /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastSwiftUpdateCheck = 1130;
+				LastUpgradeCheck = 1130;
+				ORGANIZATIONNAME = OpenCV;
+				TargetAttributes = {
+					5BBDB485240D380F00BB10FE = {
+						CreatedOnToolsVersion = 11.3.1;
+						LastSwiftMigration = 1130;
+					};
+				};
+			};
+			buildConfigurationList = 5BBDB481240D380F00BB10FE /* Build configuration list for PBXProject "ColorBlobDetectionDynamic" */;
+			compatibilityVersion = "Xcode 9.3";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+				Base,
+			);
+			mainGroup = 5BBDB47D240D380F00BB10FE;
+			productRefGroup = 5BBDB487240D380F00BB10FE /* Products */;
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				5BBDB485240D380F00BB10FE /* ColorBlobDetection */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXResourcesBuildPhase section */
+		5BBDB484240D380F00BB10FE /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				5BBDB496240D381000BB10FE /* LaunchScreen.storyboard in Resources */,
+				5BBDB493240D381000BB10FE /* Assets.xcassets in Resources */,
+				5BBDB491240D380F00BB10FE /* Main.storyboard in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXSourcesBuildPhase section */
+		5BBDB482240D380F00BB10FE /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				5BBDB48E240D380F00BB10FE /* ViewController.swift in Sources */,
+				5B8C4C31243849E9003D9176 /* ColorBlobDetector.swift in Sources */,
+				5BBDB48A240D380F00BB10FE /* AppDelegate.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin PBXVariantGroup section */
+		5BBDB48F240D380F00BB10FE /* Main.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				5BBDB490240D380F00BB10FE /* Base */,
+			);
+			name = Main.storyboard;
+			sourceTree = "<group>";
+		};
+		5BBDB494240D381000BB10FE /* LaunchScreen.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				5BBDB495240D381000BB10FE /* Base */,
+			);
+			name = LaunchScreen.storyboard;
+			sourceTree = "<group>";
+		};
+/* End PBXVariantGroup section */
+
+/* Begin XCBuildConfiguration section */
+		5BBDB4AE240D381000BB10FE /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_EMPTY_BODY = YES;
+				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
+				CLANG_WARN_INT_CONVERSION = YES;
+				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
+				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_TESTABILITY = YES;
+				FRAMEWORK_SEARCH_PATHS = ../..;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				GCC_DYNAMIC_NO_PIC = NO;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"DEBUG=1",
+					"$(inherited)",
+				);
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = "../../OpenCV/**";
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
+				MTL_FAST_MATH = YES;
+				ONLY_ACTIVE_ARCH = YES;
+				OTHER_LDFLAGS = "";
+				SDKROOT = iphoneos;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_OBJC_BRIDGING_HEADER = "";
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+			};
+			name = Debug;
+		};
+		5BBDB4AF240D381000BB10FE /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_EMPTY_BODY = YES;
+				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
+				CLANG_WARN_INT_CONVERSION = YES;
+				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
+				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
+				ENABLE_NS_ASSERTIONS = NO;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				FRAMEWORK_SEARCH_PATHS = ../..;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = "../../OpenCV/**";
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				MTL_ENABLE_DEBUG_INFO = NO;
+				MTL_FAST_MATH = YES;
+				OTHER_LDFLAGS = "";
+				SDKROOT = iphoneos;
+				SWIFT_COMPILATION_MODE = wholemodule;
+				SWIFT_OBJC_BRIDGING_HEADER = "";
+				SWIFT_OPTIMIZATION_LEVEL = "-O";
+				VALIDATE_PRODUCT = YES;
+			};
+			name = Release;
+		};
+		5BBDB4B1240D381000BB10FE /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = NO;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/../..",
+				);
+				INFOPLIST_FILE = ColorBlobDetection/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				OTHER_SWIFT_FLAGS = "-Xcc -Wno-incomplete-umbrella";
+				PRODUCT_BUNDLE_IDENTIFIER = org.opencv.ColorBlobDetection;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
+				SWIFT_OBJC_BRIDGING_HEADER = "";
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
+			};
+			name = Debug;
+		};
+		5BBDB4B2240D381000BB10FE /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = NO;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/../..",
+				);
+				INFOPLIST_FILE = ColorBlobDetection/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				OTHER_SWIFT_FLAGS = "-Xcc -Wno-incomplete-umbrella";
+				PRODUCT_BUNDLE_IDENTIFIER = org.opencv.ColorBlobDetection;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
+				SWIFT_OBJC_BRIDGING_HEADER = "";
+				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
+			};
+			name = Release;
+		};
+/* End XCBuildConfiguration section */
+
+/* Begin XCConfigurationList section */
+		5BBDB481240D380F00BB10FE /* Build configuration list for PBXProject "ColorBlobDetectionDynamic" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				5BBDB4AE240D381000BB10FE /* Debug */,
+				5BBDB4AF240D381000BB10FE /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+		5BBDB4B0240D381000BB10FE /* Build configuration list for PBXNativeTarget "ColorBlobDetection" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				5BBDB4B1240D381000BB10FE /* Debug */,
+				5BBDB4B2240D381000BB10FE /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+/* End XCConfigurationList section */
+	};
+	rootObject = 5BBDB47E240D380F00BB10FE /* Project object */;
+}

--- a/samples/swift/ios/ColorBlobDetection/ColorBlobDetectionDynamic.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
+++ b/samples/swift/ios/ColorBlobDetection/ColorBlobDetectionDynamic.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
@@ -1,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>IDEDidComputeMac32BitWarning</key>
+    <true/>
+</dict>
+</plist>

--- a/samples/swift/ios/FaceDetection/FaceDetectionDynamic.xcodeproj/project.pbxproj
+++ b/samples/swift/ios/FaceDetection/FaceDetectionDynamic.xcodeproj/project.pbxproj
@@ -1,0 +1,423 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 50;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		5B79BD652439FF020062DD99 /* DetectionBasedTracker.mm in Sources */ = {isa = PBXBuildFile; fileRef = 5B79BD642439FF020062DD99 /* DetectionBasedTracker.mm */; };
+		5B8C4C2D243814E9003D9176 /* libc++.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = 5B8C4C2C243814A0003D9176 /* libc++.tbd */; };
+		5B8C4C2F2438383D003D9176 /* lbpcascade_frontalface.xml in Resources */ = {isa = PBXBuildFile; fileRef = 5B8C4C2E2438383D003D9176 /* lbpcascade_frontalface.xml */; };
+		5BBDB48A240D380F00BB10FE /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5BBDB489240D380F00BB10FE /* AppDelegate.swift */; };
+		5BBDB48E240D380F00BB10FE /* ViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5BBDB48D240D380F00BB10FE /* ViewController.swift */; };
+		5BBDB491240D380F00BB10FE /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 5BBDB48F240D380F00BB10FE /* Main.storyboard */; };
+		5BBDB493240D381000BB10FE /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 5BBDB492240D381000BB10FE /* Assets.xcassets */; };
+		5BBDB496240D381000BB10FE /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 5BBDB494240D381000BB10FE /* LaunchScreen.storyboard */; };
+		5BBDB517241923C800BB10FE /* AVFoundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5BBDB516241923C800BB10FE /* AVFoundation.framework */; };
+		5BBDB5192419241200BB10FE /* CoreMedia.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5BBDB5182419241200BB10FE /* CoreMedia.framework */; };
+		5BE7205A25201437004E84AE /* OpenCV.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5B14CF812452EED3003438BE /* OpenCV.framework */; };
+		5BE7205B25201437004E84AE /* OpenCV.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 5B14CF812452EED3003438BE /* OpenCV.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+/* End PBXBuildFile section */
+
+/* Begin PBXCopyFilesBuildPhase section */
+		5BE7205C25201437004E84AE /* Embed Frameworks */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				5BE7205B25201437004E84AE /* OpenCV.framework in Embed Frameworks */,
+			);
+			name = "Embed Frameworks";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXCopyFilesBuildPhase section */
+
+/* Begin PBXFileReference section */
+		5B14CF812452EED3003438BE /* OpenCV.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = OpenCV.framework; path = ../../../OpenCV.framework; sourceTree = "<group>"; };
+		5B79BD622439FEB60062DD99 /* DetectionBasedTracker.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = DetectionBasedTracker.h; sourceTree = "<group>"; };
+		5B79BD632439FF020062DD99 /* FaceDetection-Bridging-Header.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "FaceDetection-Bridging-Header.h"; sourceTree = "<group>"; };
+		5B79BD642439FF020062DD99 /* DetectionBasedTracker.mm */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.objcpp; path = DetectionBasedTracker.mm; sourceTree = "<group>"; };
+		5B8C4C2C243814A0003D9176 /* libc++.tbd */ = {isa = PBXFileReference; lastKnownFileType = "sourcecode.text-based-dylib-definition"; name = "libc++.tbd"; path = "usr/lib/libc++.tbd"; sourceTree = SDKROOT; };
+		5B8C4C2E2438383D003D9176 /* lbpcascade_frontalface.xml */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xml; path = lbpcascade_frontalface.xml; sourceTree = "<group>"; };
+		5BBDB486240D380F00BB10FE /* FaceDetection.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = FaceDetection.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		5BBDB489240D380F00BB10FE /* AppDelegate.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AppDelegate.swift; sourceTree = "<group>"; };
+		5BBDB48D240D380F00BB10FE /* ViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ViewController.swift; sourceTree = "<group>"; };
+		5BBDB490240D380F00BB10FE /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/Main.storyboard; sourceTree = "<group>"; };
+		5BBDB492240D381000BB10FE /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
+		5BBDB495240D381000BB10FE /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
+		5BBDB497240D381000BB10FE /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
+		5BBDB516241923C800BB10FE /* AVFoundation.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = AVFoundation.framework; path = System/Library/Frameworks/AVFoundation.framework; sourceTree = SDKROOT; };
+		5BBDB5182419241200BB10FE /* CoreMedia.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = CoreMedia.framework; path = System/Library/Frameworks/CoreMedia.framework; sourceTree = SDKROOT; };
+/* End PBXFileReference section */
+
+/* Begin PBXFrameworksBuildPhase section */
+		5BBDB483240D380F00BB10FE /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				5BBDB5192419241200BB10FE /* CoreMedia.framework in Frameworks */,
+				5B8C4C2D243814E9003D9176 /* libc++.tbd in Frameworks */,
+				5BBDB517241923C800BB10FE /* AVFoundation.framework in Frameworks */,
+				5BE7205A25201437004E84AE /* OpenCV.framework in Frameworks */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXFrameworksBuildPhase section */
+
+/* Begin PBXGroup section */
+		5BBDB47D240D380F00BB10FE = {
+			isa = PBXGroup;
+			children = (
+				5BBDB488240D380F00BB10FE /* FaceDetection */,
+				5BBDB487240D380F00BB10FE /* Products */,
+				5BBDB515241923C800BB10FE /* Frameworks */,
+			);
+			sourceTree = "<group>";
+		};
+		5BBDB487240D380F00BB10FE /* Products */ = {
+			isa = PBXGroup;
+			children = (
+				5BBDB486240D380F00BB10FE /* FaceDetection.app */,
+			);
+			name = Products;
+			sourceTree = "<group>";
+		};
+		5BBDB488240D380F00BB10FE /* FaceDetection */ = {
+			isa = PBXGroup;
+			children = (
+				5B14CF812452EED3003438BE /* OpenCV.framework */,
+				5B8C4C2E2438383D003D9176 /* lbpcascade_frontalface.xml */,
+				5BBDB489240D380F00BB10FE /* AppDelegate.swift */,
+				5BBDB48D240D380F00BB10FE /* ViewController.swift */,
+				5BBDB48F240D380F00BB10FE /* Main.storyboard */,
+				5BBDB492240D381000BB10FE /* Assets.xcassets */,
+				5BBDB494240D381000BB10FE /* LaunchScreen.storyboard */,
+				5BBDB497240D381000BB10FE /* Info.plist */,
+				5B79BD622439FEB60062DD99 /* DetectionBasedTracker.h */,
+				5B79BD642439FF020062DD99 /* DetectionBasedTracker.mm */,
+				5B79BD632439FF020062DD99 /* FaceDetection-Bridging-Header.h */,
+			);
+			path = FaceDetection;
+			sourceTree = "<group>";
+		};
+		5BBDB515241923C800BB10FE /* Frameworks */ = {
+			isa = PBXGroup;
+			children = (
+				5B8C4C2C243814A0003D9176 /* libc++.tbd */,
+				5BBDB5182419241200BB10FE /* CoreMedia.framework */,
+				5BBDB516241923C800BB10FE /* AVFoundation.framework */,
+			);
+			name = Frameworks;
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXNativeTarget section */
+		5BBDB485240D380F00BB10FE /* FaceDetection */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 5BBDB4B0240D381000BB10FE /* Build configuration list for PBXNativeTarget "FaceDetection" */;
+			buildPhases = (
+				5BBDB482240D380F00BB10FE /* Sources */,
+				5BBDB483240D380F00BB10FE /* Frameworks */,
+				5BBDB484240D380F00BB10FE /* Resources */,
+				5BE7205C25201437004E84AE /* Embed Frameworks */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = FaceDetection;
+			productName = FaceDetection;
+			productReference = 5BBDB486240D380F00BB10FE /* FaceDetection.app */;
+			productType = "com.apple.product-type.application";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		5BBDB47E240D380F00BB10FE /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastSwiftUpdateCheck = 1130;
+				LastUpgradeCheck = 1130;
+				ORGANIZATIONNAME = OpenCV;
+				TargetAttributes = {
+					5BBDB485240D380F00BB10FE = {
+						CreatedOnToolsVersion = 11.3.1;
+						LastSwiftMigration = 1130;
+					};
+				};
+			};
+			buildConfigurationList = 5BBDB481240D380F00BB10FE /* Build configuration list for PBXProject "FaceDetectionDynamic" */;
+			compatibilityVersion = "Xcode 9.3";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+				Base,
+			);
+			mainGroup = 5BBDB47D240D380F00BB10FE;
+			productRefGroup = 5BBDB487240D380F00BB10FE /* Products */;
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				5BBDB485240D380F00BB10FE /* FaceDetection */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXResourcesBuildPhase section */
+		5BBDB484240D380F00BB10FE /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				5BBDB496240D381000BB10FE /* LaunchScreen.storyboard in Resources */,
+				5BBDB493240D381000BB10FE /* Assets.xcassets in Resources */,
+				5BBDB491240D380F00BB10FE /* Main.storyboard in Resources */,
+				5B8C4C2F2438383D003D9176 /* lbpcascade_frontalface.xml in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXSourcesBuildPhase section */
+		5BBDB482240D380F00BB10FE /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				5B79BD652439FF020062DD99 /* DetectionBasedTracker.mm in Sources */,
+				5BBDB48E240D380F00BB10FE /* ViewController.swift in Sources */,
+				5BBDB48A240D380F00BB10FE /* AppDelegate.swift in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXSourcesBuildPhase section */
+
+/* Begin PBXVariantGroup section */
+		5BBDB48F240D380F00BB10FE /* Main.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				5BBDB490240D380F00BB10FE /* Base */,
+			);
+			name = Main.storyboard;
+			sourceTree = "<group>";
+		};
+		5BBDB494240D381000BB10FE /* LaunchScreen.storyboard */ = {
+			isa = PBXVariantGroup;
+			children = (
+				5BBDB495240D381000BB10FE /* Base */,
+			);
+			name = LaunchScreen.storyboard;
+			sourceTree = "<group>";
+		};
+/* End PBXVariantGroup section */
+
+/* Begin XCBuildConfiguration section */
+		5BBDB4AE240D381000BB10FE /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_EMPTY_BODY = YES;
+				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
+				CLANG_WARN_INT_CONVERSION = YES;
+				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
+				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_TESTABILITY = YES;
+				FRAMEWORK_SEARCH_PATHS = ../..;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				GCC_DYNAMIC_NO_PIC = NO;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"DEBUG=1",
+					"$(inherited)",
+				);
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = "../../OpenCV/**";
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
+				MTL_FAST_MATH = YES;
+				ONLY_ACTIVE_ARCH = YES;
+				OTHER_LDFLAGS = "";
+				SDKROOT = iphoneos;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_OBJC_BRIDGING_HEADER = "FaceDetection/FaceDetection-Bridging-Header.h";
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+			};
+			name = Debug;
+		};
+		5BBDB4AF240D381000BB10FE /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
+				CLANG_CXX_LIBRARY = "libc++";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_BOOL_CONVERSION = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_EMPTY_BODY = YES;
+				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
+				CLANG_WARN_INT_CONVERSION = YES;
+				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
+				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
+				ENABLE_NS_ASSERTIONS = NO;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				FRAMEWORK_SEARCH_PATHS = ../..;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				HEADER_SEARCH_PATHS = "../../OpenCV/**";
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				MTL_ENABLE_DEBUG_INFO = NO;
+				MTL_FAST_MATH = YES;
+				OTHER_LDFLAGS = "";
+				SDKROOT = iphoneos;
+				SWIFT_COMPILATION_MODE = wholemodule;
+				SWIFT_OBJC_BRIDGING_HEADER = "FaceDetection/FaceDetection-Bridging-Header.h";
+				SWIFT_OPTIMIZATION_LEVEL = "-O";
+				VALIDATE_PRODUCT = YES;
+			};
+			name = Release;
+		};
+		5BBDB4B1240D381000BB10FE /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = NO;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/../..",
+				);
+				HEADER_SEARCH_PATHS = "$(PROJECT_DIR)/../../OpenCV.framework/Headers";
+				INFOPLIST_FILE = FaceDetection/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				OTHER_SWIFT_FLAGS = "-Xcc -Wno-incomplete-umbrella";
+				PRODUCT_BUNDLE_IDENTIFIER = org.opencv.FaceDetection;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
+				SWIFT_OBJC_BRIDGING_HEADER = "FaceDetection/FaceDetection-Bridging-Header.h";
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
+			};
+			name = Debug;
+		};
+		5BBDB4B2240D381000BB10FE /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = NO;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/../..",
+				);
+				HEADER_SEARCH_PATHS = "$(PROJECT_DIR)/../../OpenCV.framework/Headers";
+				INFOPLIST_FILE = FaceDetection/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				OTHER_SWIFT_FLAGS = "-Xcc -Wno-incomplete-umbrella";
+				PRODUCT_BUNDLE_IDENTIFIER = org.opencv.FaceDetection;
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
+				SWIFT_OBJC_BRIDGING_HEADER = "FaceDetection/FaceDetection-Bridging-Header.h";
+				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
+			};
+			name = Release;
+		};
+/* End XCBuildConfiguration section */
+
+/* Begin XCConfigurationList section */
+		5BBDB481240D380F00BB10FE /* Build configuration list for PBXProject "FaceDetectionDynamic" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				5BBDB4AE240D381000BB10FE /* Debug */,
+				5BBDB4AF240D381000BB10FE /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+		5BBDB4B0240D381000BB10FE /* Build configuration list for PBXNativeTarget "FaceDetection" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				5BBDB4B1240D381000BB10FE /* Debug */,
+				5BBDB4B2240D381000BB10FE /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+/* End XCConfigurationList section */
+	};
+	rootObject = 5BBDB47E240D380F00BB10FE /* Project object */;
+}

--- a/samples/swift/ios/FaceDetection/FaceDetectionDynamic.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
+++ b/samples/swift/ios/FaceDetection/FaceDetectionDynamic.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
@@ -1,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>IDEDidComputeMac32BitWarning</key>
+    <true/>
+</dict>
+</plist>

